module/People/src/People/Controller/IndexController.php

```diff
@@ -19,4 +22,22 @@ public function organizationsAction()
 	{
 
 	}
+	
+	public function profileAction()
+	{
+		$organizationService = $this->getServiceLocator()->get('People\OrganizationService');
+		$organization = $organizationService->findOrganization($this->params('orgId'));
+		if (is_null($organization)){
+			$this->response->setStatusCode(404);
+		}
+		if(is_null($this->params('id'))){
+			$this->response->setStatusCode(400);
+		}
+				
+		$userService = $this->getServiceLocator()->get('Application\UserService');
+		$user = $userService->findUser($this->params('id'));
```

Comment:
Perché carichi l'utente? Le sue info vengono caricate in modo asincrono chiamando l'API `user-profile`